### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Arcoders**

### CODE REVIEW

File: staff/ismael-haytam/js-arrays/for-each.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function forEach(arr, callback) {
+
+    // validation
+    if (!(arr instanceof Array)) throw Error(arr + ' is not an array');
+    if (typeof callback !== 'function') throw Error(callback + ' is not a function');
+
+    // logic
+    for (var i = 0; !!(arr[i]); i++) callback(arr[i], i, arr);
```
Comment (maryammdot) [inline staff/ismael-haytam/js-arrays/for-each.js:8]: Cool logic inside for!!

File: staff/ismael-haytam/js-arrays/index-of.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+function indexOf(arr, elem) {
+    
+    // validation
+    if (!(arr instanceof Array))  throw Error("first parameter should be an array!")
+    
+    if (!arr.length || !elem.length)  throw Error("one argument is empty");
+
+    // logic
+    for (var i = 0; i < arr.length; i++) if (arr[i] == elem) return i;
```
Comment (maryammdot) [inline staff/ismael-haytam/js-arrays/index-of.js:9]: Maybe better === instead of == ?

File: staff/ismael-haytam/js-arrays/for-each.js (hunk 1 above)
Comment (Arcoders) [reply staff/ismael-haytam/js-arrays/for-each.js:8]: thanks ^^

File: staff/ismael-haytam/js-arrays/index-of.js (hunk 2 above)
Comment (Arcoders) [reply staff/ismael-haytam/js-arrays/index-of.js:9]: you're right

**Pull request merged**